radixdlt-engine/src/main/java/com/radixdlt/engine/RadixEngine.java

```diff
@@ -400,6 +400,7 @@ private REParsedTxn verify(CMStore.Transaction dbTransaction, Txn txn, Permissio
 			try {
 				this.engineStore.storeTxn(dbTransaction, txn, parsedTxn.stateUpdates());
 			} catch (Exception e) {
+				logger.error("Store of atom failed. Cause: {}", e);
```

Comment:
I don't think this works since last parameter is throwable. Would need to do `e.toString()`

Reply:
Updated